topics/climate/tutorials/pangeo-notebook/tutorial.md

```diff
@@ -0,0 +1,723 @@
+as coordinates (if they have some), all the attributes such as the name, the physical units, etc.
+
+## Select / Subset from coordinates
+
+We often want to select elements from the coordinates for instance to subset a geographical area or select specific times or a specific time range.
+
+There are two different ways to select:
+- by index
+- by value
+
+### Select elements from coordinate by index
+
+```python
+print(dset.isel(time=0))
+```
+
+You should see that the coordinate `time` "disappeared" from the `Dimensions` and now the variable `pm2p5_conc` is a 3D field with longitude, latitude and level.
+
+### Select elements from coordinates by value
+
+When selecting elements by the value of the coordinate, we need to use the same datatype. For instance, to select and element from
+`time`, we need to use `timedelta64`. The code below will give the same result as `isel(time=0)`.
+
+```python
+print(dset.sel(time=np.timedelta64(0)))
+```
+
+The output will be very similar to what we did previously when selecting from coordinates by index.
+
+> ### {% icon question %} Select a single time for PM2.5
+>
+> How to select the forecast for December, 24th 2021 at 12:00 UTC?
+>
+> > ### {% icon solution %} Solution
+> > Data starts on December, 22nd 2021 at 00:00 UTC so we need to add 2 days and 12 hours to select the correct time index.
+> >
+> > > ### {% icon code-in %} Input: Python
+> > > ```python
+> > > print(dset.sel(time=(np.timedelta64(2,'D')+ np.timedelta64(12,'h'))))
+> > > ```
+> > {: .code-in}
+> > 
+> > > ### {% icon code-out %} Output
+> > > ```bash
+> > > <xarray.Dataset>
+> > > Dimensions:     (longitude: 700, latitude: 400, level: 1)
+> > > Coordinates:
+> > >   * longitude   (longitude) float32 -24.95 -24.85 -24.75 ... 44.75 44.85 44.95
+> > >   * latitude    (latitude) float32 69.95 69.85 69.75 69.65 ... 30.25 30.15 30.05
+> > >   * level       (level) float32 0.0
+> > >     time        timedelta64[ns] 2 days 12:00:00
+> > > Data variables:
+> > >     pm2p5_conc  (level, latitude, longitude) float32 0.4499 0.4421 ... 10.71
+> > > Attributes:
+> > >     title:        PM25 Air Pollutant FORECAST at the Surface
+> > >     institution:  Data produced by Meteo France
+> > >     source:       Data from ENSEMBLE model
+> > >     history:      Model ENSEMBLE FORECAST
+> > >     FORECAST:     Europe, 20211222+[0H_96H]
+> > >     summary:      ENSEMBLE model hourly FORECAST of PM25 concentration at the...
+> > >     project:      MACC-RAQ (http://macc-raq.gmes-atmosphere.eu)
+> > > ```
+> > {: .code-out}
+> {: .solution }
+{: .question }
+
+
+## Plotting
+
+- To plot a map, you need to select a variable with data on geographical coordinates (latitude, longitude).
+- In addition coordinates need to be sorted, and preferably in increasing order. This is not the case for the coordinate "longitude" which is given between 360 and 0. Let's shift the longitudes by 180 degrees so that they come in the range of -180 to 180.
```

Comment:
```suggestion
To plot a map, you need to select a variable with data on geographical coordinates (latitude, longitude). In addition, coordinates need to be sorted, and preferably in increasing order. This is not the case for the coordinate "longitude" which is given between 360 and 0. 

Let's shift the longitudes by 180 degrees so that they come in the range of -180 to 180.
```